demo/src/public/experiment-page2.js

```diff
@@ -1,14 +1,34 @@
 (function() {
     'use strict';
 
-    var lou = new Mozilla.TrafficCop({
+    var variants = {
+        'a': 30,
+        'b': 30,
+        'c': 30
+    };
+
+    function handleVariation(variation) {
+        // wait until DOM is ready to be manipulated...
+        domReady(function() {
+            // make sure variation is one we are expecting (and not noVariationCookieValue)
+            if (variants.hasOwnProperty(variation)) {
+                var target = document.getElementById('var-' + variation);
+                target.classList.remove('hidden');
+            }
+        });
+    }
+
+    // non-jquery document ready function
+    // http://beeker.io/jquery-document-ready-equivalent-vanilla-javascript
+    function domReady(callback) {
```

Comment:
Hmm, I'm a little torn on this. Ideally, I don't think we should be running traffic cop tests on browsers that don't get 1st class CSS/JS as it will likely just skew our data. But I know we're not quite there yet on bedrock. At the same time, 99% of people will be using Traffic Cop in the head of their pages since this is what the docs suggest. Nearly all of the time, I expect some form of DOM ready handler will be needed. Even if we did supply one, people could monkey patch their own implementation should they need one? I understand trying to keep the lib light, but at the same time I think any alternative library that did this would handle when to execute code. I'm happy to go with what you think is best / for now though.